pom.xml

```diff
@@ -191,6 +191,22 @@
       <activation>
         <activeByDefault>false</activeByDefault>
       </activation>
+      <modules>
+        <module>gluten-data</module>
+        <module>gluten-celeborn</module>
+        <module>backends-velox</module>
+      </modules>
+      <properties>
+        <backend_type>velox</backend_type>
+        <arrow.version>11.0.0-gluten</arrow.version>
+        <build_velox_backend>ON</build_velox_backend>
+      </properties>
+    </profile>
+    <profile>
+      <id>backends-velox-rssoff</id>
```

Comment:
Can we just set the id as "backends-velox-rss" and set to true by default for better reading since it may be hard to read rssoff and set to false

Reply:
@weiting-chen done